language/stdlib/modules/RegisteredCurrencies.move

```diff
@@ -15,34 +15,23 @@ module RegisteredCurrencies {
         currency_codes: vector<vector<u8>>,
     }
 
-    const ENOT_GENESIS: u64 = 0;
     const EINVALID_SINGLETON_ADDRESS: u64 = 1;
```

Comment:
nit: shift these numbers down by one to make sequential starting at 0

Reply:
Done